aiida/engine/processes/workchains/restart.py

```diff
@@ -29,7 +30,7 @@ class BaseRestartWorkChain(WorkChain):
 
     This work chain implements the most basic functionality to achieve this goal. It will launch the sub process,
     restarting until it is completed successfully or the maximum number of iterations is reached. It can recover from
-    errors through error handlers that can be registered to the class through the `register_process_handler` decorator.
+    errors through error handlers that can be registered to the class through the `process_handler` decorator.
```

Comment:
> handlers that can be registered to the class

I feel that 'registered to the class' is still referring to the previous approach. One should stress that handlers should be part of a class.

Reply:
Agree, will change it